types/node/v4/index.d.ts

```diff
@@ -1,6 +1,8 @@
 // Type definitions for Node.js 4.2
 // Project: http://nodejs.org/
-// Definitions by: Microsoft TypeScript <http://typescriptlang.org>, DefinitelyTyped <https://github.com/DefinitelyTyped/DefinitelyTyped>
+// Definitions by: Microsoft TypeScript <http://typescriptlang.org>,
+//                 DefinitelyTyped <https://github.com/DefinitelyTyped/DefinitelyTyped>
+//                 Marc Ghorayeb <https://github.com/marcghorayeb>
```

Comment:
This change is needed? Type definition has *many* contributors and I don't think that all of them should be listed here.